react/features/etherpad/middleware.js

```diff
@@ -23,9 +25,10 @@ MiddlewareRegistry.register(({ dispatch, getState }) => next => action => {
     switch (action.type) {
     case TOGGLE_DOCUMENT_EDITING: {
         if (typeof APP === 'undefined') {
-            const { editing } = getState()['features/etherpad'];
+            const editing = !getState()['features/etherpad'].editing;
 
-            dispatch(setDocumentEditingState(!editing));
+            dispatch(setDocumentEditingState(editing));
+            dispatch(setActiveModalId(editing ? SHARE_DOCUMENT_VIEW_ID : undefined));
```

Comment:
this looks like it would hide another visible moda, is thatt what we want?

Reply:
good point, thanks!